test/unit/rate_estimate_test.rb

```diff
@@ -58,4 +58,8 @@ def test_delivery_category_is_set
     assert_equal "local_delivery", est.delivery_category
   end
 
+  def test_delivery_date_pulls_from_delivery_range
+    assert_equal [DateTime.new(2016, 7, 1), DateTime.new(2016, 7, 3)], @rate_estimate.delivery_range
+    assert_equal DateTime.new(2016, 7, 3), @rate_estimate.delivery_date
```

Comment:
Test `delivery_date` as being `delivery_range.last` in all cases, meaning we don't have to test `delivery_date` in other tests. Makes the tests simpler and removes duplication.